hypha/static_src/src/app/src/containers/ReminderContainer/index.js

```diff
@@ -0,0 +1,128 @@
+            this.props.initAction(this.props.submissionID)
+        }
+    }
+
+    handleModalClose = () => {
+        this.setState({open : false})
+    }
+
+    render(){
+        const { classes } = this.props;
+        return (
+            this.props.reminderInfo.loading 
+            ? <LoadingPanel /> :
+            <div className="reminder-container">
+                <SidebarBlock title={"Reminders"}>
+                    <div className="status-actions">
+                        <button 
+                            className="button button--primary button--half-width button--bottom-space reminder-button" 
+                            onClick={() => this.setState({open : true})}
+                        >
+                            Create Reminder
+                        </button>
+                        <Modal
+                            className={classes.modal} 
+                            open={this.state.open}
+                        >
+                            <>
+                            <ReminderForm 
+                                submissionID={this.props.submissionID} 
+                                closeForm={() => this.setState({open: false})}
+                            />
+                            </>
+                        </Modal>
+                        {this.props.reminders && this.props.reminders.length
+                        ?
+                        this.props.reminders.map(reminders =>
+                        <ReminderList
+                            key={reminders.grouper}
+                            title={reminders.grouper}
+                            reminders={reminders.list} 
+                            submissionID={this.props.submissionID} 
+                            deleteReminder={this.props.deleteReminder}
+                        />)
+                        :
+                        <div>No reminders yet.</div>}
+                    </div>
+                </SidebarBlock>
+            </div>
+        )
+    }
+}
+
+ReminderContainer.propTypes = {
+    reminderInfo: PropTypes.object,
+    initAction: PropTypes.func,
+    deleteReminder: PropTypes.func,
+    classes: PropTypes.object,
+    submissionID: PropTypes.number,
+    reminders: PropTypes.array
+}
+
+const mapStateToProps = state =>  ({
+    reminderInfo : Selectors.selectReminderContainer(state),
+    reminders: Selectors.selectReminders(state)
+});
+  
+  
+function mapDispatchToProps(dispatch) {
+    return bindActionCreators({
+        initAction: Actions.initializeAction,
+        deleteReminder: Actions.deleteReminderAction,
+    },
+    dispatch,
+    );
+}
+  
+const withConnect = connect(
+    mapStateToProps,
+    mapDispatchToProps,
+);
+  
+const withReducer = injectReducer({ key: 'ReminderContainer', reducer });
+const withSaga = injectSaga({ key: 'ReminderContainer', saga });
+  
+  
+export default compose(
+withSaga,
+withReducer,
+withConnect,
+withRouter,
+withStyles(styles)
```

Comment:
Style nitpick:

These lines should be indented.

```suggestion
    withSaga,
    withReducer,
    withConnect,
    withRouter,
    withStyles(styles)
```